mlflow/store/abstract_store.py

```diff
@@ -195,20 +202,46 @@ def get_metric_history(self, run_id, metric_key):
         """
         pass
 
-    @abstractmethod
     def search_runs(self, experiment_ids, filter_string, run_view_type,
-                    max_results=SEARCH_MAX_RESULTS_DEFAULT, order_by=None):
+                    max_results=SEARCH_MAX_RESULTS_DEFAULT, order_by=None, page_token=None):
```

Comment:
@mparkhe is `None` an okay default value for the page token (based on how you plan to implement validation/handling of this argument in the server)?

Reply:
Actually looks like proto won't send the field if it's `None` so should be fine

Reply:
👍